legislators-social-media.yaml

```diff
@@ -4175,7 +4175,7 @@
     twitter: RepSarbanes
     facebook: RepSarbanes
     twitter_id: 364415553
-    youtube_id: list=UUCypi0IDTr-XxlAXsJCdBZQ
+    youtube_id: UUCypi0IDTr-XxlAXsJCdBZQ
```

Comment:
Hmm, actually this doesn't seem to resolve to a valid channel:

https://www.youtube.com/channel/UUCypi0IDTr-XxlAXsJCdBZQ

So perhaps it would be better to remove this one entirely, @pullyl.